src/openrct2/network/NetworkBase.cpp

```diff
@@ -1698,15 +1698,23 @@ bool NetworkBase::ProcessConnection(NetworkConnection& connection)
 void NetworkBase::ProcessPacket(NetworkConnection& connection, NetworkPacket& packet)
 {
     const auto& handlerList = GetMode() == NETWORK_MODE_SERVER ? server_command_handlers : client_command_handlers;
-    auto it = handlerList.find(packet.GetCommand());
-    if (it != handlerList.end())
+
+    try
```

Comment:
Sounds like the scope of the `try` block can be made smaller, as I'm assuming the `find` and the `if`s there wouldn't really throw.